plugins/speculation-rules/hooks.php

```diff
@@ -12,6 +12,53 @@
 }
 // @codeCoverageIgnoreEnd
 
+/**
+ * Determines whether Speculative Loading is enabled.
+ *
+ * @since n.e.x.t
+ *
+ * @return bool Whether enabled.
+ */
+function plsr_is_speculative_loading_enabled(): bool {
+	$option = plsr_get_stored_setting_value();
+
+	// Disabled if the user is logged in, unless the setting explicitly allows the current user's role.
+	if (
+		( is_user_logged_in() )
```

Comment:
Ah, these extra parentheses can be removed.

```suggestion
		is_user_logged_in()
```